app/parsers/ports.py

```diff
@@ -0,0 +1,18 @@
+from app.objects.secondclass.c_relationship import Relationship
+from app.utility.base_parser import BaseParser
+
+
+class Parser(BaseParser):
+
+    def parse(self, blob):
+        relationships = []
+        for match in self.load_json(blob):
```

Comment:
this may not be worthwhile-- but do you think there's value in making this a generic parser, for core app/learning/*? in our current parsing world, there are use-cases for both specific parsers tied to ability YML files (like this) as well as generic parsers that will run against any ability output.

Reply:
I think we need to implement quite a large number of things in the learning_svc... to include more parsers and requirements enforcing.